### DISCUSSION TITLE
Automatically use "default" config across all repos in user/org

### DISCUSSION THREAD

**Discussion by fregante** [renovatebot/renovate#24453] [Suggest an Idea]
### Tell us more.

GitHub has a special `user/.github` **repository** where users and organizations can put their shared files. All the other repos in the user or organization will automatically inherit some of these special files, like `funding.yml`, unless a new file is created in a specific repository.

More info: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

I'd love to see Renovate automatically configure **all** my repositories using this shared config either in this special `user/.github` repo or in its own `user/.renovate` one, so I don't have to manually create a `renovate.json` file in every repo. 

The closest solution to this is to use shared configurations, but that still requires creating a `renovate.json` in each repository.

**Comment by rarkins** [comment]
When onboarding a repo, Renovate checks for the file `default.json` in e.g. `github.com/fregante/renovate-config`. If found then the onboarding config will automatically extend that preset instead of the bot's default.

If that's not found then it looks for `renovate-config.json` in e.g. `github.com/fregante/.github` and extends that instead.

In other words, at onboarding time, Renovate will add an explicit extends from each repo to your shared config.

We don't yet have an _implicit_ extends as you're asking for, but there's a feature request issue here: #22212

**Reply by fregante** [reply]
> it looks for `renovate-config.json` in e.g. `github.com/fregante/.github` and extends that instead.

Not a bad start!

> We don't yet have an _implicit_ extends as you're asking for, but there's a feature request issue here: #22212

Sounds like what I'm asking, yeah. Essentially I want to enable renovate on my entire account and just receive PRs on schedule, without further onboarding/dashboard noise.

I don't know if it's practical on pre-existing users, but it might be on new organizations.

**Reply by viceice** [reply]
this is currently only possible by @rarkins. he can add a special global config for your organization/ user to apply to all repos in that scope.

**Reply by rarkins** [reply]
Yes, good point, although it's no longer me. The team which runs the app can for now manually provision default config per org on request

**Reply by viceice** [reply]
ok, didn't know that. so you now fully concentrate on renovate OSS development (with features for hosted app/ on prem)?

**Reply by rarkins** [reply]
Essentially yes. I'm nonlingwr involved with _operations_ much
